proxy/fluentd/README.md

```diff
@@ -130,4 +130,90 @@ fluent-gem install rusty_json_schema
 fluent-gem install fluent-plugin-out-http
 ```
 Once the external dependencies are installed, a single Ruby code file `parser_openlineage.rb` needs
-to be copied into the Fluentd plugins directory ([installing custom plugin](https://docs.fluentd.org/plugin-development#installing-custom-plugins)).
\ No newline at end of file
+to be copied into the Fluentd plugins directory ([installing custom plugin](https://docs.fluentd.org/plugin-development#installing-custom-plugins)).
+
+## Workin with Prometheus
```

Comment:
Could you rephrase this title, so that it clearly mentions this relates to Fluent setup? 
Can be something like:
```
## Fluentd proxy setup
### Monitoring with Prometheus
```

Reply:
Thank you for your feedback. I've just made the changes you asked for and uploaded them here. Could you please have a look and see if there's anything else to be changed?